bin/logstash.lib.sh

```diff
@@ -72,12 +72,33 @@ parse_jvm_options() {
   fi
 }
 
+setup_bundled_jdk_part() {
+  if [ "$OSTYPE" = "darwin"* ]; then
```

Comment:
```suggestion
  if [[ "$OSTYPE" = "darwin"* ]]; then
```

Requires double brackets for the wildcard test to work

Reply:
The `[[ ... ]]` is a bash syntax, the problem is that `logstash.lib.sh` is included both in `#!/bin/sh` scripts and `#!/bin/bash` so we can't use bash specific idioms. I switched to `uname -s`  and sh string comparison:
```sh
OS_NAME="$(uname -s)"
if [ $OS_NAME = "Darwin" ]; then
 ...
fi
```